app/main.py

```diff
@@ -1,3 +1,10 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    count = 0
```

Comment:
The initial assignment of `count = 0` is correct, but it should be aligned with the rest of the code for better readability. Consider moving it to line 3 to maintain consistent indentation.